package.json

```diff
@@ -39,6 +41,7 @@
   "devDependencies": {
     "@babel/core": "^7.6.4",
     "@jest/core": "26.6.3",
+    "@react-three/fiber": "^7.0.26",
```

Comment:
Needing to add this to the root package.json is probably a symptom of something else we were doing, and likely no longer required thanks to the update to the bootstrap command.